daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -1,12 +1,16 @@
 ---
 type: docs
-title: "Azure Events Hub"
-linkTitle: "Azure Events Hub"
-description: "Detailed documentation on the Azure Event Hubs pubsub component"
+title: "Azure Events Hub (deprecated)"
```

Comment:
Suggestion to remove the "s". "Azure Events Hub" => "Azure Event Hub"

Reply:
Absolutely, I just appended "deprecated" to the misspelled name here.

Reply:
```suggestion
title: "Azure Event Hubs (deprecated)"
```